doc/src/getting-started-guide.md

```diff
@@ -316,13 +316,13 @@ standard libraries (`atomvmlib.uf2`).
 
 ## Getting Started on the Generic UNIX platform
 
-AtomVM may be run on UNIX-like platforms using the `atomvm` command.
+AtomVM may be run on UNIX-like platforms using the `AtomVM` command.
 
```

Comment:
Actually, it is lower-case.  When you run make install, a script is installed (lower case) which includes the core atomvmlibs automatically.  This will be the basis for a Homebrew formula, once we do a release (because we need a version for that)

Reply:
Thank you for the info.